### ISSUE TITLE
Travis Fails for some of the PRs

### ISSUE THREAD

**Issue by AndrewLister-STFC** [fitbenchmarking/fitbenchmarking#259]
**Description of the error**
2 of the PRs in the queue are failing with no information.

**Describe the expected result**
Failures should be informative and travis should reflect running on local machines

**Describe the actual result**
See above

**Comment by AndrewLister-STFC** [comment]
I have emailed Travis support to get debug access to investigate further.

**Comment by AndrewLister-STFC** [comment]
Copy from #226 
================================

Debugging this looks like its a hdf5/mantid communication issue.

Mantid is compiled with hdf5 ver 1.8.16, whereas the version pip installs is 1.10.4. So it fails with a warning.

```
/usr/local/lib/python2.7/dist-packages/h5py-2.10.0-py2.7-linux-x86_64.egg/h5py/__init__.py:40: UserWarning: h5py is running against HDF5 1.8.16 when it was built against 1.10.4, this may cause problems
  '{0}.{1}.{2}'.format(*version.hdf5_built_version_tuple)
Warning! ***HDF5 library version mismatched error***
The HDF5 header files used to compile this application do not match
the version used by the HDF5 library to which this application is linked.
Data corruption or segmentation faults may occur if the application continues.
This can happen when an application was compiled by one version of HDF5 but
linked with a different version of static or shared HDF5 library.
You should recompile the application or check your shared library related
settings such as 'LD_LIBRARY_PATH'.
You can, at your own risk, disable this warning by setting the environment
variable 'HDF5_DISABLE_VERSION_CHECK' to a value of '1'.
Setting it to 2 or higher will suppress the warning messages totally.
Headers are 1.10.4, library is 1.8.16
```

**Comment by AndrewLister-STFC** [comment]
Fixed by #226
